config/config.exs

```diff
@@ -1,3 +1,3 @@
 use Mix.Config
 
-config :grpc, http2_client_adapter: GRPC.Adapter.Gun
+config :grpc, http2_client_adapter: GRPC.Client.Adapters.Gun
```

Comment:
I like how this looks :)

We need to deprecate the need for application env config, though, but that's another issue